saleor/graphql/payment/types.py

```diff
@@ -323,6 +325,14 @@ def resolve_external_url(root: models.TransactionEvent, info):
     def resolve_name(root: models.TransactionEvent, info):
         return root.message
 
+    @staticmethod
+    def resolve_created_by(root: models.TransactionItem, info):
```

Comment:
Shouldn't be wrapped in any permissions?

Reply:
I've already added separate to-do step to define a proper permission for this field as I need to re-think what needs to be required here. For now, let's leave it as it is but thanks for catching this.
Ref: https://github.com/saleor/saleor/pull/10350
![image](https://user-images.githubusercontent.com/23282005/208863122-f7428420-8c4d-42bc-a90f-9aa8eb762233.png)